qcog_python_client/monitor/_wandb.py

```diff
@@ -0,0 +1,44 @@
+"""Wandb Monitor implementation."""
+
+import os
+
+import wandb
+
+from .interface import Monitor
+
+WANDB_DEFAULT_PROJECT = "qcognitive-dev"
+
+
+class WandbMonitor(Monitor):
+    """Wandb Monitor implementation."""
+
+    def init(
```

Comment:
This whole thing looks like a context manager to be honest, since you have a close call. It would be better if it was a context manager with an __enter__ and __exit__ method so that users can't mess it up by not closing things as this is public.

Reply:
I don't like context manager when the block too many lines and honestly this is the same pattern that both `wandb` and `ubiops` metrics are using so I would follow the same pattern to make integration easier